PotFieldPrediction/src/python/tests/test_edmd.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import numpy as np
+
+
+import matplotlib.pyplot as plt
+
+from PotFieldPrediction.src.python.models.edmd import EDMDModel
+
+
+def load_data_pot_fields(
```

Comment:
If you need it in both files and it does not test anything you have two reasons to put this in a `utils.py`.